include/nudb/impl/context.ipp

```diff
@@ -119,6 +119,29 @@ run()
     cv_w_.notify_all();
 }
 
+void
+context::
+flush()
+{
+    {
+        std::lock_guard<std::mutex> lock(m_);
+
+        if (stop_ || num_threads_ > 0)
+            return;
+
+        // Move everything in waiting_ to flushing_
+        for(auto store = waiting_.head_; store;
+            store = store->next_)
+            store->state_ = store_base::state::flushing;
+        flushing_.splice(waiting_);
+    }
+
+    // Flush all stores
```

Comment:
Nit: For consistency, this comment could be `// Process everything in flushing_`

Reply:
Done.